fyde-app/linux.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: default
+title: Linux
+parent: Fyde App
+nav_order: 1
+---
+# Fyde App - Linux
+
+Instructions to install the Fyde App in different distributions.
+
+There are two versions available:
+
+- fyde - complete GUI version
```

Comment:
@GMartinez-Sisti perhaps you can refer here (or somewhere below) that `fyde` includes `fyde-cli`

Reply:
@oNaiPs You mean in the repository?

Reply:
@GMartinez-Sisti apologies, I meant `fyde-tunnel`... My point is just for the users to know that if they install `fyde`, they dont need to install `fyde-tunnel` to have the daemon only interface

Reply:
Makes sense @oNaiPs

Reply:
Updated.